web/lib/client/apollo.ts

```diff
@@ -0,0 +1,57 @@
+// from https://github.com/awslabs/aws-mobile-appsync-sdk-js#creating-a-client
+
+import { ApolloClient, ApolloLink, createHttpLink, InMemoryCache } from '@apollo/client';
+import { SentryLink } from 'apollo-link-sentry';
+import { AUTH_TYPE, createAuthLink } from 'aws-appsync-auth-link';
+import { createSubscriptionHandshakeLink } from 'aws-appsync-subscription-link';
+import { getSession } from 'next-auth/react';
+import { APPSYNC_ENDPOINT, REGION } from '../config/next';
```

Comment:
I would prefer REGION to be AWS_REGION but not a problem

Reply:
ok